packages/adapters/watcher/src/config.ts

```diff
@@ -0,0 +1,78 @@
+import { existsSync, readFileSync } from "fs";
+
+import { ajv } from "@connext/nxtp-utils";
+import { Static, Type } from "@sinclair/typebox";
+
+export const WatcherConfigSchema = Type.Object({
```

Comment:
Actually all this code should be in the agent. The adapter should assume this gets passed into it and not deal with any of the actual env stuff. We can add this to the top-level watcher class and then pass it in through the agent config.

Reply:
Fixed in here
https://github.com/connext/nxtp/pull/2654